docs/UserGuide.md

```diff
@@ -190,9 +186,11 @@ Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [g/GENDER] [age/AGE
 </div>
 
 Examples:
-* `edit 1 n/Malan i/[2] Swimming` edits the name of contact and the second interest of the first contact in the displayed list
-* `edit 1 i/Painting i/[1] Running` adds 'painting' as an interest and edits the first interest of the first contact in the displayed list
+* `edit 1 n/Malan i/(2) Swimming` edits the name of contact and the second interest of the first contact in the displayed list
```

Comment:
change to "edits the name of the first contact in the displayed list to 'malan' and its second interest to 'swimming'